### PR TITLE
Attempt removal of git from pre-commit command

### PR DESCRIPTION
**Pull request by SimonBiggs**
This appears to fix https://github.com/kynan/nbstripout/issues/105

...hopefully it doesn't break it though...

### CODE REVIEW

File: .pre-commit-hooks.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 -   id: nbstripout
     name: nbstripout
     description: 'nbstripout: strip output from Jupyter and IPython notebooks'
-    entry: bash -c 'nbstripout "$@"; git add -u' --
+    entry: bash -c 'nbstripout "$@"' --
     language: python
```
Comment (kynan) [inline .pre-commit-hooks.yaml]: This is effectively the same as just `nbstripout` iiuc

Comment (SimonBiggs) [reply .pre-commit-hooks.yaml]: Yeah, in the end I removed the bash also as that made it not work on Windows. At the end of the day I think a plain `entry: nbstripout` works best.

Comment (kynan) [reply .pre-commit-hooks.yaml]: Can you please make that change and I'll merge this PR.

File: .pre-commit-hooks.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 -   id: nbstripout
     name: nbstripout
     description: 'nbstripout: strip output from Jupyter and IPython notebooks'
-    entry: bash -c 'nbstripout "$@"; git add -u' --
+    entry: bash -c 'nbstripout "$@"' --
```
Comment (kynan) [inline .pre-commit-hooks.yaml]:
```suggestion
    entry: nbstripout
```

Comment (kynan) [reply .pre-commit-hooks.yaml]: Please make this change and I'll merge this PR

Comment (SimonBiggs) [reply .pre-commit-hooks.yaml]: Thanks @kynan :)

**Comment by SimonBiggs** [comment]
Thanks @kynan :)

**Pull request merged**